.github/workflows/ci.yaml

```diff
@@ -44,14 +40,14 @@ jobs:
             #   If you change it, make sure to adjust lower bounds there to reflect the change.
             #   Also, make sure to adjust tested-with field in package.yaml so that it contains
             #   corresponding GHC version.
-            stack-resolver: lts-18.21
+            stack-resolver: lts-19.33
```

Comment:
Lower bound for testing, corresponds to GHC 9.0.2., three years old.

I picked this one so that there are versions of GHC newer than 8.10.7 that don't have an appropriate StrongPath version:
- Up to 9.0.1 - Strong Path 1.1.4.0
- After 9.0.2 - Strong path 1.2.0.0

Reply:
I didn't quite get the second part of the message (everything beyond first sentence), could you explain in simpler way?

Reply:
The sentence was non-sensical due to edits, so no wonder you couldn't understand it.

Edited :)